libkbfs/tlf_journal.go

```diff
@@ -1902,19 +1902,19 @@ func (j *tlfJournal) getMDRange(
 }
 
 func (j *tlfJournal) doPutMD(ctx context.Context, rmd *RootMetadata,
-	mdInfo unflushedPathMDInfo,
-	perRevMap unflushedPathsPerRevMap) (
-	mdID kbfsmd.ID, retryPut bool, err error) {
+	mdInfo unflushedPathMDInfo, perRevMap unflushedPathsPerRevMap,
+	verifyingKey kbfscrypto.VerifyingKey) (
```

Comment:
Yeah, we have to pass the verifying key all the way down from FBO because it could possibly come from the predecessor MD update, which isn't available easily here in the journal -- we'd have to re-fetch it from the cache/server.  Seems cleaner to pass it all the way down.  I guess I'll add a comment on the MDOps interface function description.